frontend/src/components/widgets/ActionLogCards.js

```diff
@@ -35,14 +35,12 @@ const useStyles = makeStyles((theme) => ({
     variant: "outlined",
     color: "default",
     size: "small",
-    
-    ["@media (max-width:620px)"]: {// eslint-disable-line no-useless-computed-key
+    "@media (max-width:620px)": {
       fontSize: "0.7rem",
     },
   },
   title: {
-    
-    ["@media (max-width:620px)"]: {// eslint-disable-line no-useless-computed-key
+    "@media (max-width:620px)": {
```

Comment:
Why are you turning the es-lint warnings back on? Was that intentional or were you not sure what these comments are for?

Reply:
I removed the square brackets and that fixed the warnings. https://eslint.org/docs/latest/rules/no-useless-computed-key

So I didn't think the comments were still necessary. Should I put them back?

Reply:
This one works. I'm not sure why the other one is giving an issue